infrastructure/tooling/src/changelog/index.js

```diff
@@ -132,16 +145,36 @@ class ChangeLog {
     const silentLinks = silent.map(sn => sn.reference).join(', ');
     const silentSuffix = silentCount === 0 ?
       '' :
-      `\n\n▶ Additional change${silentCount === 1 ? '' : 's'} not described here: ${silentLinks}.`;
-
-    return this.snippets
-      .filter(sn => sn.level !== 'silent')
-      .map(({level, reference, body}) => (
-        '▶ ' + levelLabels[level] +
-        (reference ? reference : '') + '\n' +
-        body.trim()
-      ))
-      .join('\n\n') + silentSuffix;
+      `\n\n### OTHER\n\n▶ Additional change${silentCount === 1 ? '' : 's'} not described here: ${silentLinks}.`;
+
+    // These snippets are already sorted in level-order so when we insert
+    // them here they remain in order. no need to re-sort
+    const categorizedSnippets = this.snippets.reduce((acc, {audience, ...rest}) => {
```

Comment:
What is snippet here?

Reply:
snippets are the text we write describing the changelog. I'll try to come up with a better name or add a comment